src/test/resources/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs/multiallelicQualRegression.expected.vcf

```diff
@@ -56,26 +56,26 @@
+chr1	3836468	.	GTTTT	GTTTTT,GTTTTTTT,G,GTTTTTT	881.67	.	AC=3,2,1,3;AF=0.026,0.018,8.772e-03,0.026;AN=114;BaseQRankSum=0.00;DP=2100;ExcessHet=0.1247;FS=2.272;InbreedingCoeff=0.1698;MLEAC=2,2,1,3;MLEAF=0.018,0.018,8.772e-03,0.026;MQ=54.73;MQRankSum=0.00;QD=10.37;ReadPosRankSum=-2.300e-01;SOR=0.483	GT:AD:DP:GQ:PL	0/1:11,22,0,0,0:44:99:239,0,174,338,219,636,338,219,636,636,338,219,636,636,636	0/0:27,0,0,0,0:27:50	0/0:30,0,0,0,0:30:10	0/0:28,0,0,0,0:28:10	./.:3,0,0,0,0:3:0	0/0:27,0,0,0,0:27:60	0/0:59,0,0,0,0:59:30	0/2:3,0,7,0,0:10:59:132,142,223,0,80,59,142,223,80,223,142,223,80,223,223	0/0:33,0,0,0,0:33:60	0/0:31,0,0,0,0:31:60	.:0,0,0,0,0	0/0:27,0,0,0,0:27:60	0/0:30,0,0,0,0:30:60	0/0:51,0,0,0,0:51:40	0/0:29,0,0,0,0:29:60	0/0:31,0,0,0,0:31:60	0/3:8,0,0,4,0:14:99:103,160,534,160,534,534,0,437,437,670,160,534,534,437,534	0/0:30,0,0,0,0:30:60	0/0:32,0,0,0,0:32:60	0/0:37,0,0,0,0:37:60	0/0:49,0,0,0,0:49:40	0/0:36,0,0,0,0:36:60	0/2:2,0,2,0,0:4:34:34,42,113,0,71,65,42,113,71,113,42,113,71,113,113	0/0:10,0,0,0,0:14:29:0,29,176,29,176,176,29,176,176,176,29,176,176,176,176	0/0:58,0,0,0,0:58:30	0/0:32,0,0,0,0:32:60	0/0:11,0,0,0,0:13:34:0,34,285,34,285,285,34,285,285,285,34,285,285,285,285	0/0:26,0,0,0,0:26:60	0/0:26,0,0,0,0:26:60	0/0:32,0,0,0,0:32:60	.:0,0,0,0,0	0/0:29,0,0,0,0:29:60	0/4:1,0,0,0,6:11:35:94,175,487,175,487,487,175,487,487,487,0,90,90,90,35	0/0:40,0,0,0,0:40:10	0/0:41,0,0,0,0:41:60	0/0:30,0,0,0,0:30:60	0/4:2,0,0,0,9:11:10:186,193,234,193,234,234,193,234,234,234,0,41,41,41,10	0/0:43,0,0,0,0:43:20	0/4:0,0,0,0,6:8:11:123,146,198,146,198,198,146,198,198,198,0,24,24,24,11	0/0:28,0,0,0,0:28:10	0/0:29,0,0,0,0:29:60	.:0,0,0,0,0	0/0:30,0,0,0,0:30:60	0/0:10,0,0,0,0:14:88:0,88,622,88,622,622,88,622,622,622,88,622,622,622,622	0/0:34,0,0,0,0:34:30	0/0:23,0,0,0,0:23:50	0/0:37,0,0,0,0:37:50	0/0:29,0,0,0,0:29:60	0/0:34,0,0,0,0:34:30	0/0:43,0,0,0,0:43:20	.:0,0,0,0,0	0/0:31,0,0,0,0:31:60	0/0:39,0,0,0,0:39:60	0/0:28,0,0,0,0:28:60	0/0:36,0,0,0,0:36:60	0/0:38,0,0,0,0:38:60	0/0:36,0,0,0,0:36:30	0/0:31,0,0,0,0:31:50	0/0:29,0,0,0,0:29:50	.:0,0,0,0,0	0/0:33,0,0,0,0:33:60	.:0,0,0,0,0	1/1:0,2,0,0,0:2:8:47,8,0,47,8,47,47,8,47,47,47,8,47,47,47	0/0:15,0,0,0,0:15:10	.:0,0,0,0,0
```

Comment:
QUAL changed, so QD changed.  I'm honestly not sure why QUAL changed when the PLs are all the same... I think the new one is correct because the old QUAL was the same in the previous PR with AN=116